src/Services/Cda/CdaTemplateParse.php

```diff
@@ -97,7 +97,9 @@ public function parseCDAEntryComponents($components): array
                     }
                 }
             }
-            $this->templateData = $postParseEvent->getTemplateData();
+            if (!empty($postParseEvent)) {
```

Comment:
Does empty() function work on objects?

Reply:
It'd be safer to check the instance type.  Empty will work, but if someone returns a string or something other than an object having a getTemplateData() method this will blow up
```
if ($postParseEvent instanceof CDAPostParseEvent)
```